pkg/cmd/setup_apps.go

```diff
@@ -77,9 +77,16 @@ var appsCmd = &cobra.Command{
 			}
 		}
 
-		//push the app config to cluster
-		//prepare agent proto to push app config
-		//agent store data on cassandra
+		client, err := agent.GetAgentClient(captenConfig)
+		if err != nil {
+			clog.Logger.Errorf("failed to fetch client, err:%v", err)
+			return
+		}
+
+		if err := agent.SaveAppConfigsOnAgent(client, captenConfig.AppsTempDirPath); err != nil {
+			clog.Logger.Errorf("Err while SaveAppConfigsOnAgent: %v", err)
```

Comment:
pass captenConfig object to method, it need to build the complete path

Reply:
done